pkg/frame/rgb_test.go

```diff
@@ -0,0 +1,48 @@
+package frame
+
+import (
+	"fmt"
+	"testing"
+)
+
```

Comment:
It would be nice to have a simple test like: https://github.com/pion/mediadevices/blob/dafd208de79bcf3308e4ea9e5724ad27f91c5051/pkg/frame/yuv_test.go#L10-L41